Gemfile

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+source "https://rubygems.org"
+
+gem "rexml"
```

Comment:
We'd probably want to use nokogiri if we pushed this to "production" as it's the more commonly used, but this does the job alright in this context.